local-volume/helm/provisioner/values.yaml

```diff
@@ -103,3 +103,12 @@ daemonset:
   #  nodeLabels:
   #    - failure-domain.beta.kubernetes.io/zone
   #    - failure-domain.beta.kubernetes.io/region
+  #
+  # If configured, tolerations will add a toleration field to the DaemonSet PodSpec.
+  #
+  #  tolerations:
+  #   - key: "{Key is the taint key that the toleration applies to.}"
+  #     operator: Equal|Exist
+  #     value: "{Value is the taint value the toleration matches to. If the operator is Exists, the value should be empty, otherwise just a regular string.}"
+  #     effect: NoSchedule|PreferNoSchedule|NoExecution
+  #     tolerationSeconds: <optional, by default not set>
```

Comment:
Because toleration is a spec of kubernetes api, add a reference to its docs will be better.

```
  # Node tolerations for local-volume-provisioner scheduling to nodes with taints.
  # Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
  tolerations: []
    # - key: "key"
    #   operator: "Equal|Exists"
    #   value: "{Value is the taint value the toleration matches to. If the operator is Exists, the value should be empty, otherwise just a regular string.}"
    #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
    #   tolerationSeconds: "{Optional field that dictates how long the pod will stay bound to the node after the taint is added.}"
```

Reply:
Thank you for that input, it does make more sense :) Fixed.